site/management.md

```diff
@@ -395,6 +402,22 @@ For instance, if you configured the CSP header with the value `default-src 'self
 
 In addition to the `connect-src` CSP header, RabbitMQ also needs the CSP directives `unsafe-eval` `unsafe-inline`, otherwise the OAuth 2.0 functionality may not work.
 
+### Identity-Provider initiated logon
+
+By default, the Management UI uses OAuth 2.0 authorization code flow to authenticate and authorize users.
+However, there are scenarios where users preferred to be automatically redirected to RabbitMQ without getting
```

Comment:
add "RabbitMQ" before the words "Management UI" in the first sentence just to be clear what UI it is at all times.